spec/requests/registrations_spec.rb

```diff
@@ -1,56 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I think we should not delete this file, but restore the master version of it so it stays unchanged

Reply:
This changes make this spec fails, so I just removed it. This spec will be handled in this [PR](https://github.com/jackrootstrap/rs-target/pull/7)